subscriptions/package.json

```diff
@@ -18,7 +18,7 @@
     "start:webpack": "npm run build:webpack -- --watch"
   },
   "dependencies": {
-    "pubnub": "4.21.6"
+    "pubnub": "^4.21.6"
```

Comment:
Update it to current one. We've had issues when some pubnub versions were not working properly, that's why we set certain version, not a wildcard.